.github/workflows/markdown-links.yml

```diff
@@ -1,23 +1,20 @@
 name: Check Markdown links
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches: [main]
+  schedule:
+    # 2am UTC daily
+    - cron: '0 2 * * *'
 
 jobs:
   markdown-link-check:
+    strategy:
+      matrix:
+        branch: ['main', 'v0.37.x', 'v0.34.x']
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: technote-space/get-diff-action@v6
         with:
-          PATTERNS: |
-            **/**.md
+          ref: ${{ matrix.branch }}
       - uses: creachadair/github-action-markdown-link-check@master
```

Comment:
Should we consider using the upstream version instead?

Reply:
I've switched this out with our own fork now: https://github.com/informalsystems/github-action-markdown-link-check